documentation/asciidoc/accessories/camera/gstreamer.adoc

```diff
@@ -0,0 +1,44 @@
+=== Using Gstreamer
+
+_Gstreamer_ is a Linux framework for reading, processing and playing multimedia files. There is a lot of information and many tutorials at the https://gstreamer.freedesktop.org/[_gstreamer_ website]. Here we show how `libcamera-vid` (and similarly `raspivid`) can be used to stream video over a network.
```

Comment:
Only a minor niggle but it seems a bit inconsistent to mention "stream video over a network" here, but all the examples only send and receive via `localhost` ? Perhaps it's worth including at least one example where the server and client are different computers?